test/e2e/deploy/deploy.go

```diff
@@ -915,6 +932,23 @@ func findDriver(c *Cluster) (*Deployment, error) {
 }
 
 func findOperator(c *Cluster) (*Deployment, error) {
+	// We have to try multiple times because the Deployment
+	// controller might still be working on the ReplicaSets that
+	// findOperatorOnce is based on.
+	for i := 0; ; i++ {
+		deployment, err := findOperatorOnce(c)
+		if err == nil {
+			return deployment, nil
+		}
+		if i > 6 {
+			return nil, err
+		}
+		framework.Logf("finding operator failed during attempt #%d, will retry: %v", err)
+		time.Sleep(10 * time.Second)
```

Comment:
Sounds like 10 seconds retry timeout might result in a delay. I think we could retry after 1 or 2 seconds.

Reply:
Changed to one second.